README.org

```diff
@@ -59,6 +61,35 @@ You can verify if log4slime is enabled, by looking for the Log4CL menu
 on top menubar in REPL and Lisp mode buffers, or by submitting a log
 statement from REPL and seeing output colorized.
 
+** Sly support
+
+As of version 1.1.4, Log4Slime supports Sly as well as Slime.  The
+minimal supported Emacs version is now 25.1.
+
+To enable Log4Sly run the following in your Lisp:
+
+#+BEGIN_SRC common-lisp
+(ql:quickload :log4ly)
```

Comment:
Typo in `log4ly`?

Reply:
Also, isn't the system name `log4cl.log4sly`?

Reply:
Yes.  Fixed.